Disable autodetection in configure, but also disable detection fallback
of both SYS_getrandom and __NR_getrandom to avoid accidental enabling
in lib/randutils.c
--- a/configure.ac
+++ b/configure.ac
@@ -462,7 +462,6 @@ AC_CHECK_FUNCS([ \
 	getdtablesize \
 	getexecname \
 	getmntinfo \
-	getrandom \
 	getrlimit \
 	getsgnam \
 	inotify_init \
--- a/lib/randutils.c
+++ b/lib/randutils.c
@@ -27,14 +27,14 @@
 
 #ifdef HAVE_GETRANDOM
 # include <sys/random.h>
-#elif defined (__linux__)
+#elif 0
 # if !defined(SYS_getrandom) && defined(__NR_getrandom)
    /* usable kernel-headers, but old glibc-headers */
 #  define SYS_getrandom __NR_getrandom
 # endif
 #endif
 
-#if !defined(HAVE_GETRANDOM) && defined(SYS_getrandom)
+#if 0
 /* libc without function, but we have syscal */
 #define GRND_NONBLOCK 0x01
 #define GRND_RANDOM 0x02
